aries-site/src/examples/templates/persistent-navigation/HeaderNavExample.js

```diff
@@ -0,0 +1,269 @@
+  },
+  {
+    name: 'Flows',
+    icon: <Splits />,
+    content: 'list',
+  },
+  {
+    name: 'Locations',
+    icon: <Location />,
+    content: 'list',
+  },
+];
+
+export const HeaderNavExample = ({ mobile }) => {
+  const [activeItem, setActiveItem] = React.useState(1);
+
+  return (
+    <ResponsiveContext.Provider value={mobile && 'small'}>
+      <ResponsiveContext.Consumer>
+        {size => (
+          <AppContainer activeItem={activeItem}>
+            <Header fill="horizontal" pad="medium">
+              <AppIdentity name={pages[activeItem].name} />
+              <Box direction="row" gap="medium">
+                <MainNavigation
+                  activeItem={activeItem}
+                  setActiveItem={setActiveItem}
+                />
+                {size !== 'small' && (
+                  <Stack anchor="bottom-right">
+                    <Avatar background="blue!">DS</Avatar>
+                    <Box
+                      background="status-ok"
+                      pad={{ horizontal: 'xsmall' }}
+                      height="12px"
+                      width="12px"
+                      round
+                    />
+                  </Stack>
+                )}
+              </Box>
+            </Header>
+            <PageContent
+              gridArea="pageContent"
+              activeItem={activeItem}
+              setActiveItem={setActiveItem}
+            />
+          </AppContainer>
+        )}
+      </ResponsiveContext.Consumer>
+    </ResponsiveContext.Provider>
+  );
+};
+
+HeaderNavExample.propTypes = {
+  mobile: PropTypes.bool,
+};
+
+const AppContainer = ({ ...rest }) => {
+  const size = React.useContext(ResponsiveContext);
+
+  return (
+    <Box
+      background="background-back"
+      height={size === 'small' ? { max: 'large' } : undefined}
+      width={size === 'small' ? 'medium' : '100%'}
+      overflow="auto"
+      {...rest}
+    />
+  );
+};
+
+const MainNavigation = ({ activeItem, setActiveItem }) => {
+  const size = React.useContext(ResponsiveContext);
+
+  return size !== 'small' ? (
+    <Nav direction="row" gap="xsmall">
+      <NavItems activeItem={activeItem} setActiveItem={setActiveItem} />
+    </Nav>
+  ) : (
+    <Menu
+      icon={<MenuIcon />}
+      dropAlign={{ top: 'bottom', right: 'right' }}
+      hoverIndicator
+      items={pages.map((item, index) => ({
+        label: item.name,
+        onClick: () => setActiveItem(index),
+      }))}
+    />
+  );
+};
+
+MainNavigation.propTypes = {
+  activeItem: PropTypes.number,
+  setActiveItem: PropTypes.func,
+};
+
+const NavItems = ({ activeItem, setActiveItem }) => {
+  return (
+    pages &&
+    pages.map((item, index) => (
+      <SimpleButton
+        key={item.name}
+        name={item.name}
+        active={index === activeItem}
+        onClick={() => setActiveItem(index)}
+        round="xsmall"
+      />
+    ))
+  );
+};
+
+export const SimpleButton = ({ active, name, ...rest }) => {
```

Comment:
I dont think you. need to export this since your within the file

Reply:
good call. will fix.

Reply:
@britt6612 This should be resolved